lab/lab2/js/part3-application.js

```diff
@@ -42,14 +46,38 @@ var getAndParseData = function() {
   /* =====================
     Fill out this function definition
   ===================== */
+  var Crimedata = "http://raw.githubusercontent.com/CPLN692-MUSA611/datasets/master/json/philadelphia-crime-snippet.json";
+  var y = $.ajax(Crimedata).done(function(y) {
+    myData = JSON.parse(y);
+  });
 };
 
 /* =====================
   Call our plotData function. It should plot all the markers that meet our criteria (whatever that
   criteria happens to be — that's entirely up to you)
 ===================== */
+
+/*=====================
+    Fill out this function definition
+  =====================*/
+console.log(newMarkers);*/
+var markerfilter = function(y) {
+  var markerfilterd = [];
+  for (i = 0; i < y.length; i++) {
```

Comment:
Are you able to add points to the map with this filter?

What parameters do you set?